packages/assets/build/_header-search.scss

```diff
@@ -0,0 +1,78 @@
+@use "sass:meta";
+
+@use "00-base/configure" as *;
+
+@use "01-atoms/button-search" as *;
+
+.ma__header-search {
+  
+  &__wrapper {
+  	display: flex;
+    position: relative;
+
+    .ma__button-icon {
+      border-right: none;
+    }
+
+    &--responsive {
+
+      @media ($bp-medium-max) {
+
+        .ma__header-search__pre-filter, .ma__header-search__post-filter {
+          display: none;
+        }
+      }
+    }
+  }
+
+  .ma__form {
+    display: flex;
+  }
+
+  //overwrite needs to port back to shared scss
+  
+  .ma__button-search {
+
+    &, &--secondary {
+      border: 2px solid !important; //override header-search button search border styles
+      position: relative !important;
+      margin-left: -2px !important;
+    }
+
+    & {
+
+      @include ma-button-search($c-primary-alt, $c-white);
+    }
+
+    &--secondary {
+
+      @include ma-button-search($c-white, $c-primary-alt);
+    }
+  }
+
+
+  &__input {
+    padding-right: 0 !important;
+  }
+
+
+  .ma__select-box {
+    margin-left: -2px;
+
+    &__field {
+      height:46px;
+    }
+
+    &__link {
+      height: 100%;
+      display: flex;
+      align-items: center;
+    }
+
+    label {
+      position: absolute !important;
+    }
+  }
+}
+
+@include meta.load-css("02-molecules/header-search");
```

Comment:
why is this scss loaded at the bottom?

Reply:
CSS overrides needed it to be placed there.